wrappers/Julia/CoolProp.jl

```diff
@@ -55,6 +55,7 @@ end
 
 # CoolProp::get_parameter_information_string
 function get_parameter_information_string(Key::AbstractString)
+message_buffer[1:4] = UInt8[108 111 110 103] # "long" written in UInt8
```

Comment:
A better way would be to have two arguments in this wrapper, than to convert that argument to a UInt8 UTF8 string, than to place it inside the buffer.
It is necessary to place it in the buffer as the buffer is used to take the result string back.

I did not did it as I've just explained because I didn't find yet the function to make that conversion (I need to have a UInt8 array, and the functions I found until now keep being of type string, even if internally, they are a compatible UInt8 array).
So I put this workaround that allows the function to be still working.